lib/ControlledVocab/tests/ControlledVocab-test.js

```diff
@@ -20,6 +20,24 @@ describe('ControlledVocab', () => {
   const cv = EditableList();
   const firstRow = EditableListRow();
 
+  describe('render CV when translations prop is provided', () => {
+    setupApplication();
+
+    // eslint-disable-next-line no-undef
+    beforeEach(() => mountComponent(false, server));
+
+    it('should have row count 5', () => cv.has({ rowCount: 5 }));
```

Comment:
what does this test do in context of testing translations used when translations prop is provided or not? IMO nothing

Reply:
1. It is ensuring that the component is mounting as expected when new props are provided and labelSingular prop is not.
2. rest of the tests in the same suite, ensure that the translations are picked.